clang/docs/StandardCPlusPlusModules.rst

```diff
@@ -398,6 +398,13 @@ BMIs cannot be shipped in an archive to create a module library. Instead, the
 BMIs(``*.pcm``) are compiled into object files(``*.o``) and those object files
 are added to the archive instead.
 
+clang-cl
+~~~~~~~~
+
+``clang-cl`` supports the same options as ``clang++`` for modules as detailed above;
+there is no need to prefix these options with ``/clang:``. Note that ``cl.exe``
+options to emit `IFC files <https://devblogs.microsoft.com/cppblog/using-cpp-modules-in-msvc-from-the-command-line-part-1/>` are *not* supported. The precompiled modules are also not compatible for use with ``cl.exe``.
```

Comment:
I'd like to add a statement to give suggestion for build systems:

It may be something like (You can reword it)

```
The build systems are suggested to use the option here for `clang-cl` instead of `cl.exe`.
```